inject/titus-inject-metadataproxy.c

```diff
@@ -126,7 +126,14 @@ static void setup_veth(int private_ns_fd, int container_ns_fd) {
 	rtnl_link_set_name(peer, IN_CONTAINER_INTERFACE_NAME);
 	rtnl_link_set_ns_fd(peer, container_ns_fd);
 
-	err = rtnl_link_add(nls, veth, NLM_F_CREATE | NLM_F_EXCL);
+	for (int i = 0; i < 5; i++) {
+		err = rtnl_link_add(nls, veth, NLM_F_CREATE | NLM_F_EXCL);
+		if (err == 0) {
+			break;
+		}
+		WARN_ON(err < 0, "%s: unable to add link", nl_geterror(err));
```

Comment:
Please add retrying + retry count.

Reply:
Yes please. Otherwise we'll have to like, count the number of times we see this for a task to know if it had to retry.
Or it could just say "Failed on the 5th attempt: permission denied"